VTEX - Ad Network API.json

```diff
@@ -627,7 +627,7 @@
             ],
             "post": {
                 "summary": "Send ad events",
-                "description": "Sends ad events, including impressions, views, and clicks, according to these rules:\r\n\r\n* **Impressions:** Must be sent when the ad is rendered in the interface.\r\n\r\n**Views:** Must be sent when the ad appears more than 50% on the screen for more than 1s.\r\n\r\n**Click:** Must be sent when the ad is clicked.\r\n\r\n>⚠️ The `MacId` and the `SessionID` are required and must be handled as described in the table below. They are used as anonymous identifiers to link events with orders. These attributes are the merchant's responsibility, and they must be managed at the device level.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+                "description": "Sends ad events, including impressions, views, and clicks, according to these rules:\r\n\r\n* **Impressions:** Must be sent when the ad is rendered in the interface.\r\n\r\n* **Views:** Must be sent when the ad appears more than 50% on the screen for more than 1s.\r\n\r\n* **Click:** Must be sent when the ad is clicked.\r\n\r\n>⚠️ The `MacId` and the `SessionID` are required and must be handled as described in the table below. They are used as anonymous identifiers to link events with orders. These attributes are the merchant's responsibility, and they must be managed at the device level.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Is the bold format supposed to be just for the `Views` description indeed?

Reply:
Impressions, Views and Clicks are bold:

<img width="327" alt="image" src="https://github.com/user-attachments/assets/4700fc83-2115-4a68-a118-7b15a888e2b8" />